javascript-loops/main.js

```diff
@@ -0,0 +1,65 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  var i = 0;
+  while (i < 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+    i++;
+  }
+  return numbers;
+}
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  var i = 0;
```

Comment:
Please only use currentNumber to iterate through the loop, an additional 'i' variable shouldn't be needed here.